src/olympia/abuse/models.py

```diff
@@ -370,6 +370,7 @@ class AbuseReport(ModelBase):
         ('SIGNED', 10, 'Signed'),
         ('SYSTEM', 11, 'System'),
         ('PRIVILEGED', 12, 'Privileged'),
+        ('NOT_REQUIRED', 13, 'Not required'),
```

Comment:
I am going to update some code in Firefox to report `NOT_REQUIRED` in the future so that's why this is added here.